pandasai/__init__.py

```diff
@@ -202,6 +205,8 @@ def run(
 
         self.log(f"Running PandasAI with {self._llm.type} LLM...")
 
+        self._prompt_id = str(uuid.uuid4())
```

Comment:
I would rather declare the process_id in the `__init__`. We could also have a combination of `prompt_id` and `process_id`. What do you think?

In case we want to keep also the prompt_id, let's print it so that in verbose mode we keep track of it!